docs/pages/text.js

```diff
@@ -197,6 +206,63 @@ export default function TextPage({ generatedDocGen }: {| generatedDocGen: DocGen
           `}
           />
         </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Title"
+          description={`The \`title\` attribute on a \`<div>\` can be used to show the full text of a truncated string on hover. That attribute is populated automatically when the text is truncated using \`lineClamp\`, as long as \`children\` is a string.
+           If \`children\` is a \`React.Node\` (e.g. [when using Link](/link#Link-and-Text)), use the \`title\` prop to manually set the \`title\` attribute.`}
+        >
+          <MainSection.Card
+            defaultCode={`
+<Flex alignItems="center" direction="column" gap={2}>
```

Comment:
Seems like this could use some formatting 😅

Reply:
whoa, Prettier killed my pretty formatting! 🙀

Reply:
!! strange!